travel-agency/routes/web.php

```diff
@@ -12,7 +13,7 @@
 | be assigned to the "web" middleware group. Make something great!
 |
 */
-
-Route::get('/', function () {
-    return view('welcome');
-});
+Route::get('/', [CityController::class, 'index'])->name('home');
```

Comment:
You could create a route group that utilizes the `CityController`. Read more about it [here](https://laravel.com/docs/10.x/routing#route-group-controllers).